planner/views.py

```diff
@@ -17,16 +17,15 @@ def get_date(req_day):
 def planner(request):
     if not 'user_id' in request.session.keys():
         return redirect('/')
-    context = {
-        'events': Event.objects.all()
-    }
-    # Month
     d = get_date(request.GET.get('day', None))
     cal = Calendar(d.year, d.month)
     cal.setfirstweekday(6)
-    context['cal'] = cal.whole_month(withyear=True)
-    context['week'] = cal.whole_week(d.day)
-
+    context = {
+        'events': Event.objects.all(),
```

Comment:
Do you want to show all events here or only the user's events? If the latter, you will need to change this to `Event.objects.filter(created_by = current_user)` (a variable you probably want to set just below line 19 with `current_user = User.objects.get(id = request.session[user_id])`.

Reply:
Yes, once I have the formatting down Ill be able to set it to just the users events, in the meantime keeping it like this so I have more to work with